packages/utils/scaffold.ts

```diff
@@ -45,16 +68,16 @@ export default function runTransform(transformConfig: TransformConfig, action: s
 
 	webpackConfig.forEach(
 		(scaffoldPiece: string): Promise<void> => {
-			const config: Config = transformConfig[scaffoldPiece];
+			let config: Config = transformConfig[scaffoldPiece];
```

Comment:
Config should be immutable/const here as we don't want to modify a presupplied config

Reply:
I was updating config later using `mergeHandler`, that's why this change. See the comment above for more info on my view on this now.

Reply:
Updated